src/main/java/uk/gov/hmcts/reform/divorce/orchestration/workflows/SubmitDnCaseWorkflow.java

```diff
@@ -23,11 +24,15 @@
     @Autowired
     private SubmitDnCase submitDnCase;
 
+    @Autowired
+    private DuplicateCaseValidationTask duplicateCaseValidationTask;
+
     public Map<String, Object> run(Map<String, Object> payload,
                                    String authToken,
                                    String caseId) throws WorkflowException {
         return this.execute(
             new Task[] {
+                duplicateCaseValidationTask,
```

Comment:
This is not duplicated case, this scenario is (Co)Respondent already answered.
```suggestion
                new Task[] {
                getCase
                alreadyAnsweredValidationTask,
```
I'd create a different validation task for this.